test/e2e/utils.go

```diff
@@ -238,12 +238,19 @@ func getDefaultEtcd(name, namespace, container, prefix string, provider TestProv
 	return etcd
 }
 
-func getDefaultMultiNodeEtcd(name, namespace, container, prefix string, provider TestProvider) *v1alpha1.Etcd {
+func getDefaultMultiNodeEtcdWithPeerTlsNotEnabled(name, namespace, container, prefix string, provider TestProvider) *v1alpha1.Etcd {
 	etcd := getDefaultEtcd(name, namespace, container, prefix, provider)
 	etcd.Spec.Replicas = multiNodeEtcdReplicas
 	return etcd
 }
 
+func getDefaultMultiNodeEtcdWithPeerTlsEnabled(name, namespace, container, prefix string, provider TestProvider) *v1alpha1.Etcd {
```

Comment:
```suggestion
func getDefaultMultiNodeEtcd(name, namespace, container, prefix string, provider TestProvider) *v1alpha1.Etcd {
```
Let it be same name `getDefaultMultiNodeEtcd` and by default with `peerTLS` enabled.